modules/oadp-deleting-cluster-resources-following-failure.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-deleting-cluster-resources-following-failure_{context}"]
+= Deleting cluster resources following a partially successful or a failed backup and restore using Data Mover
+
+If your backup and restore operation that uses Data Mover either fails or only partially succeeds, you need to clean up any `VolumeSnapshotBackup` (VSB) or `VolumeSnapshotrestores` custom resource definitions (CRDs) that exist in the application namespace, and clean up any extra resources created by these controllers.
```

Comment:
Check the CR names here. I don't think the capitalization is right / plural. Should it be `VolumeSnapshotRestore`?

```suggestion
If your backup and restore operation that uses Data Mover either fails or only partially succeeds, you must clean up any `VolumeSnapshotBackup` (VSB) or `VolumeSnapshotRestore` custom resource definitions (CRDs) that exist in the application namespace, and clean up any extra resources created by these controllers.
```